scripts/linters/pylint_extensions.py

```diff
@@ -1929,6 +1931,47 @@ def visit_functiondef(self, node):
                 'non-test-files-function-name-checker', node=node)
 
 
+class ConcatenationChecker(checkers.BaseChecker):
+    """Custom pylint checker which checks that string
+    concatenation is present or not
+    """
+
+    __implements__ = interfaces.ITokenChecker
+
+    name = 'string-concatenation'
+    priority = -1
+    msgs = {
+        'C0032': (
+            'At string %s, please prefer string interpolation over '
+            'concatenation',
+            'string-concatenation',
+            'Prefer: "My string %s" % varname to "My string " + varname.',
```

Comment:
Rephrase the 1st and 3rd message such that the 1st message explains what a user needs to do to fix the issue in the codebase, and the 3rd message should be info about this lint check i.e, what this lint check provides? See [doc](http://pylint.pycqa.org/en/latest/how_tos/custom_checkers.html) for more info on what each of it means. And treat other checkers as an example.
For eg 3rd msg could be like: `Enforce use of string interpolation over string concatenation.`

Reply:
Done